dockerfiles/unit/Dockerfile

```diff
@@ -78,3 +78,19 @@ RUN curl -so /etc/ssl/certs/fake_root.pem \
     -t "TCu,Cu,Tu" \
     -i /etc/ssl/certs/fake_root.pem \
     -d sql:/root/.pki/nssdb
+
+# install containerd
+ARG RUNC_VERSION=v1.0.0-rc8
+ARG CNI_VERSION=v0.8.2
+ARG CONTAINERD_VERSION=1.3.0
+
+ENV PATH=$PATH:/usr/local/concourse/bin
+
+RUN set -x && \
+	mkdir -p /usr/local/concourse/bin && \
```

Comment:
missing the `init` that's meant to be used as the replacement for `/tmp/gdn-init`

Reply:
https://github.com/concourse/ci/pull/253/commits/7e94b5e730b45bf3d0c7251f3ad56dd2f676191f